eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -195,4 +195,21 @@
         <Class name="com.azure.common.auth.credentials.AzureCliCredentials"/>
         <Bug pattern="UWF_FIELD_NOT_INITIALIZED_IN_CONSTRUCTOR"/>
       </Match>
+
+      <!-- The poll strategy class has no necessary to serialize or deserialize. -->
+      <Match>
+        <Class name="~com\.azure\.common\.mgmt\.(CompletedPollStrategy\$CompletedPollStrategyData|ProvisioningStatePollStrategy\$ProvisioningStatePollStrategyData)"/>
+        <Bug pattern="SE_NO_SERIALVERSIONID"/>
+      </Match>
+
+      <Match>
+        <Class name="~com\.azure\.common\.mgmt\.(CompletedPollStrategy\$CompletedPollStrategyData|ProvisioningStatePollStrategy\$ProvisioningStatePollStrategyData)"/>
+        <Bug pattern="SE_BAD_FIELD"/>
+      </Match>
```

Comment:
Can you leave a comment here about why these checks are suppressed?

Reply:
Added reasonable comments.

Reply:
Did you confirm with @anuchandy or @jianghaolu about these classes and their need to be serializable? I note that CompletedPollStrategy has a method `strategyData()` that returns `Serializable`.

Reply:
Will confirm with Anu. Jianghao said it is ok to his adjudgment, but needs confirm from @anuchandy . I will ask him to take a look as well.

Reply:
@sima-zhu @JonathanGiles, these types needs to be serializable. The use case is - 

User can start a long running operation and operations's progress can be tracked by polling. In such cases, we want to give user the ability to serialize the polling data (into DB e.g.) and deserialize it and use it to  continue the polling. You can imagine the case where application is running in a state-less micro-service. If an instance goes down another instance might need to continue from where the first instance stopped. Support for polling data serialization enables this.

Reply:
@sima-zhu Please make the changes requested by @anuchandy - thanks.